.github/docker-images/integration-tests/ubuntu/Dockerfile

```diff
@@ -58,6 +58,7 @@ RUN git clone --branch v2.13.6 https://github.com/catchorg/Catch2.git && \
 
 RUN git clone https://github.com/aws-samples/aws-iot-securetunneling-localproxy && \
 	cd aws-iot-securetunneling-localproxy && \
+    git checkout 851de4cc2b48861b835327d005000c7d0c81d11b && \
```

Comment:
why are we using this specific commit?

Reply:
We need to use a specific commit and this one is commit prior to the boost 1.81 upgrade which is not backwards compatible. Moving to a more recent commit is going to require a lot of work.